packages/coinjoin/src/types/coordinator.ts

```diff
@@ -129,23 +131,23 @@ export interface CoinjoinState {
     witnesses?: Record<number, string>;
 }
 
-export enum RoundPhase {
-    InputRegistration = 0,
-    ConnectionConfirmation = 1,
-    OutputRegistration = 2,
-    TransactionSigning = 3,
-    Ended = 4,
-}
-
-export enum EndRoundState {
-    None = 0,
-    AbortedWithError = 1,
-    AbortedNotEnoughAlices = 2,
-    TransactionBroadcastFailed = 3,
-    TransactionBroadcasted = 4,
-    NotAllAlicesSign = 5,
-    AbortedNotEnoughAlicesSigned = 6,
-}
+// export enum RoundPhase {
```

Comment:
Not needed anymore?

Reply:
right, totally forgot to remove them, rebased + focepushed